android/src/main/java/ee/forgr/capacitor_updater/DownloadService.java

```diff
@@ -1,19 +1,12 @@
-/*
- * This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at https://mozilla.org/MPL/2.0/.
```

Comment:
Please add back the header it's a requirement of the licence

Reply:
added back in 0fa27d2 !